docs/using-getting-started.asciidoc

```diff
@@ -0,0 +1,304 @@
+[[ecs-getting-started]]
+=== Getting Started
+
+ECS enables and encourages users to normalize event data in order
+to better analyze, visualize, and correlate their events. Collected events
+can be normalized at ingest time, consistently searched across indices,
+and visualized predictably. When adopting an Elastic solution, such as
+https://www.elastic.co/observability[Observability] or https://www.elastic.co/security[Security],
+all of this work is already done! Elastic provides an extensive set of
```

Comment:
```suggestion
all events will map to ECS out of the box. Elastic provides an extensive set of
```

I think it's worth being a little more explicit that "adopting ECS" is not much work with the solutions.